elixir-stuff/elmchemy/lib/XTuple.ex

```diff
@@ -0,0 +1,29 @@
+# Compiled using Elmchemy v0.0.22
+defmodule XTuple do
+  use Elmchemy
+
+  @spec first({any, any}) :: any
+  curry first/1
+  def first({fst, _}) do
+    fst
+  end
+
+  @spec second({any, any}) :: any
+  curry second/1
+  def second({_, snd}) do
+    snd
+  end
+
+  @spec map_first(any) :: any{any, any}) :: {any, any}
```

Comment:
Because of broken typespecs for functions it's not a proper spec #34 
Should be disabled with `{- flag nospec:mapFirst -}` for now